<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>711</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Main_Insert</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>321</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Data Insert</string>
      </property>
      <widget class="QPushButton" name="insertButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Insert Data Now</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Insert_Data">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>301</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>321</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Data Output System LOG</string>
      </property>
      <widget class="QTextBrowser" name="SystemOP_Browser">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>301</width>
         <height>201</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>321</width>
        <height>441</height>
       </rect>
      </property>
      <property name="title">
       <string>Now</string>
      </property>
      <widget class="QLabel" name="label_name">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>271</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS PMincho</family>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Howdy</string>
       </property>
      </widget>
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>281</width>
         <height>191</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_time">
       <property name="geometry">
        <rect>
         <x>36</x>
         <y>60</y>
         <width>261</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Currently</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Search_Output</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>641</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Search DATA</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>351</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Year / Month / Day </string>
       </property>
      </widget>
      <widget class="QPushButton" name="Search_Button">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>20</y>
         <width>91</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string>Search_Button</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>436</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_2"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_3"/>
        </item>
        <item>
         <widget class="QTimeEdit" name="timeEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>20</y>
         <width>331</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>KeyWord</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>50</y>
         <width>254</width>
         <height>20</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkBox_2">
          <property name="text">
           <string>Accept day categories</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Accept until minute</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>641</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Output</string>
      </property>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>601</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>601</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>256</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>255</width>
        <height>23</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QProgressBar" name="progressBar_search">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Wait For System information</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Info_Editor</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>661</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Data Selector Control</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>641</width>
         <height>49</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>1.Data ID from database</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Number.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>2147483647</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Download_Data">
              <property name="text">
               <string>Load Data</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>2.Path Selected :</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="txt_select">
              <property name="text">
               <string>Select .txt From Disk</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_4">
              <property name="text">
               <string>CheckConform</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="PasteButton">
              <property name="text">
               <string>Paste and ReplaceEditor</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>661</width>
        <height>341</height>
       </rect>
      </property>
      <property name="title">
       <string>Editor</string>
      </property>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="geometry">
        <rect>
         <x>23</x>
         <y>23</y>
         <width>611</width>
         <height>281</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>310</y>
         <width>191</width>
         <height>25</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QCheckBox" name="checkBox_5">
          <property name="text">
           <string>CheckConform</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="DataUpdate_Button">
          <property name="text">
           <string>Data_Update</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>310</y>
         <width>331</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>NOW</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTool_Bar">
    <property name="title">
     <string>language</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionLanguage"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuUI_Type">
    <property name="title">
     <string>UI_Type</string>
    </property>
   </widget>
   <addaction name="menuTool_Bar"/>
   <addaction name="menuUI_Type"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLanguage">
   <property name="text">
    <string>English</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
